service/config/config.go

```diff
@@ -42,6 +41,8 @@ type ServiceConfig struct {
 	LogFile          string
 	LogFileMaxSizeMb int
 	PidFile          string
+	UserLevel        bool
+	ServiceUserName  string
```

Comment:
not for now, but we can rename ServiceUserName to just 'UserName' like UserLevel